curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/65f41113845ed61e5065515d.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 65f41113845ed61e5065515d
+title: Step 17
+challengeType: 20
+dashedName: step-17
+---
+
+# --description--
+
+Modify the instantiation of `v1` and `v2` by using keyword arguments.
```

Comment:
Why?

Reply:
This is related to what happens later: enforce keyword arguments. I think we can consider to remove that part and teach it around step 69 of the password generator project

Reply:
this step is pretty sudden and confusing, the camper deserves to know what is going on. Please add at least an extra sentence to explain why we are changing everything to use keyword arguments